monitoring/osd-enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -7,7 +7,14 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 toc::[]
 
 [role="_abstract"]
-In {product-title}, a cluster administrator can enable alert routing for user-defined projects. This process consists of two general steps:
+In {product-title}, a cluster administrator can enable alert routing for user-defined projects. 
+
+[IMPORTANT]
+====
+Managing alerting rules for user-defined projects is only available in {product-title} version 4.11 and up. 
```

Comment:
IBMSG didn't exactly have guidance on "up", but they do suggest "later":
 higher   adverb
Do not use to describe versions of software or fix packs. Use later.